packages/base/core/src/less/variables/components/button.less

```diff
@@ -12,7 +12,7 @@
 @oui-button-font-size: rem-calc(16);
 @oui-button-font-weight: @oui-font-semibold;
 @oui-button-radius: @base-border-radius-medium;
-@oui-button-border-width: 0;
+@oui-button-border-width: 0px; // Need unit for calculation
```

Comment:
For what calculation ?

Reply:
In the `calc()`, just below your comment.
Have you read the description ?